src/conditional-compilation.md

```diff
@@ -7,7 +7,8 @@ r[cfg.syntax]
 > &nbsp;&nbsp; &nbsp;&nbsp; _ConfigurationOption_\
 > &nbsp;&nbsp; | _ConfigurationAll_\
 > &nbsp;&nbsp; | _ConfigurationAny_\
-> &nbsp;&nbsp; | _ConfigurationNot_
+> &nbsp;&nbsp; | _ConfigurationNot_\
+> &nbsp;&nbsp; | `true` | `false
```

Comment:
```suggestion
> &nbsp;&nbsp; | `true` | `false`
```

Reply:
Good catch, thank you. I've rebased the branch, so I squashed this change in.